lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -81,13 +87,26 @@
     };
 
     var resolveLocaleBestFit = function (locale, defaultLocale) {
-        var resolvedLocale = platform.resolveLocaleBestFit(locale);
+        var resolvedLocale = platform.localeBestFitCache.get(locale);
+        if (resolvedLocale === undefined) {
+            resolvedLocale = platform.resolveLocaleBestFit(locale);
```

Comment:
Good point. I think we should also cache the fact that we got `undefined` for a specific `locale` parameter so we don't try to resolve it again. I will update the code.